src/main/java/com/cosmian/jna/findex/structs/Location.java

```diff
@@ -0,0 +1,24 @@
+package com.cosmian.jna.findex.structs;
+
+import java.nio.charset.Charset;
+
+import com.cosmian.jna.findex.serde.Leb128ByteArray;
+
+public class Location extends Leb128ByteArray {
+
+    public Location() {
+        super();
+    }
+
+    public Location(byte[] value) {
+        super(value);
+    }
+
+    public Location(String location) {
+        this(location.getBytes(Charset.defaultCharset()));
```

Comment:
In `Keyword` you use `StandardCharsets.UTF_8`, which method is best? Being explicit about UTF8?

Reply:
This is a bug. Fixed. TXS